pkg/podgrouper/podgrouper/podgrouper.go

```diff
@@ -80,7 +80,7 @@ func (pg *podGrouper) GetPGMetadata(ctx context.Context, pod *v1.Pod, topOwner *
 	logger := log.FromContext(ctx)
 	ownerKind := metav1.GroupVersionKind(topOwner.GroupVersionKind())
 	plugin := pg.pluginsHub.GetPodGrouperPlugin(ownerKind)
-	logger.V(1).Info(fmt.Sprintf("Using %v plugin for pod.", plugin),
+	logger.V(1).Info(fmt.Sprintf("Using %v plugin for pod.", plugin.Name()),
```

Comment:
Ohh I see...
Do you think its work mandating every instance to implement Name() just for a lone line of code?

Reply:
So this also answers these two questions above, I was thinking it might be useful for troubleshooting to know which plugin was used. I can remove this log and the name method altogether. What do you think?